docs/containerd.md

```diff
@@ -30,4 +30,13 @@ containerd_registries:
     - "https://registry-1.docker.io"
 ```
 
+`containerd_registries` is ignored for pulling images when `image_command_tool=crictl`
```

Comment:
According to https://github.com/kubernetes-sigs/kubespray/issues/8375 `containerd_registries` is ignored if nerdctl, not crictl, right?

Reply:
Yes. I was too fast ...